### PR TITLE
add codefund

### PR DESCRIPTION
**Pull request by kentcdodds**
I'm setting up codefund to contribute revenue to our open collective.

### CODE REVIEW

**Comment by kentcdodds** [comment]
Whoops, one sec. Let me fix this really quick. I missed one requirement that codefund has.

**Comment by kentcdodds** [comment]
Ok, I reworked it a little bit and I'm happier with it. Took inspiration from https://nuxtjs.org/guide

Here's what it looks like (when the ad is shown as well as when it's blocked).

image

image

image

image

image

image

File: website/static/js/codefund.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  if (!post) {
+    // we only put the ad on pages that have a "post"
+    return
+  }
+  const codefundDiv = document.createElement('div')
+  codefundDiv.id = 'codefund-container'
+  codefundDiv.innerHTML = `
+    <div id="codefund">
+      <div>
+        Testing Library loads relevant ethical ads via <a href="https://codefund.io" rel="noopener noreferrer">CodeFund.io</a>.
```
Comment (afontcu) [inline website/static/js/codefund.js]: should this external links open a new tab?

File: website/static/js/codefund.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        The funds are contributed to <a href="https://opencollective.com/testing-library" rel="noopener noreferrer">our Open Collective</a>.
+        Please disable your ad blocker for testing-library.com
+      </div>
+    </div>
+  `
+  post.prepend(codefundDiv)
+
+  const scriptEl = document.createElement('script')
+  scriptEl.src = 'https://codefund.io/properties/389/funder.js'
```
Comment (coderberry) [inline website/static/js/codefund.js]: To ensure you do not receive any pre-formatted CSS, use `https://codefund.io/properties/389/funder.js?theme=unstyled`. That way you don't have to do any hard overwrites of styles.

**Comment by kentcdodds** [comment]
After considering this a little bit, and talking offline with others, I've decided that right now is not a good time to add codefund to testing-library.com.

**Pull request closed without merging**